vlplidar_ws/src/vlp_cartographer/config/cartographer_demo.lua

```diff
@@ -46,16 +32,11 @@ options = {
 
 MAP_BUILDER.use_trajectory_builder_2d = true
 
-TRAJECTORY_BUILDER_2D.min_range = 0.12
-TRAJECTORY_BUILDER_2D.max_range = 3.5
+TRAJECTORY_BUILDER_2D.min_range = 0.5
```

Comment:
Originally, the minimum range was set to 0.5 to prevent noise near the LiDAR in real world. However, I realized that I didn't use the correct minimum range from the LiDAR specifications. I will correct the minimum range based on the following reference.
- [Velodyne Puck (from Clearpath Robotics)](https://docs.clearpathrobotics.com/docs/robots/accessories/sensors/lidar_3d/velodyne_puck/#specifications)